### DISCUSSION TITLE
Setting up a Mode 3 course

### DISCUSSION THREAD

**Discussion by nitroSkirata** [Courseplay/courseplay#7259] [Q&A] [answered]
Hey,
How does one set up a Mode3 Course?

I still do it, like it was done in FS17 (Course from field, waitpoint at overloading point, corse to the field).

Has there been any changes to this setup?
Currently my drivers are waiting indefinitely, after unloading.

Kind regards,
JDTiger

**Comment by Tensuko** [comment, marked as answer]
https://youtu.be/YhB5EXQfqIE
